.python-version

```diff
@@ -0,0 +1 @@
+3.4.1
```

Comment:
I was not expecting to keep it (only if a good reason for it is presented), either way we should give preference to python 3.5.